great_expectations/data_context/store/in_memory_store_backend.py

```diff
@@ -59,7 +59,9 @@ def _get(self, key):
 
     @override
     def _get_all(self) -> list[Any]:
-        raise NotImplementedError
+        return [
+            val for key, val in self._store.items() if key != self.STORE_BACKEND_ID_KEY
+        ]
```

Comment:
Implemented to get in-memory based tests working - we should get this working for all backends but not a priority